post.md

```diff
@@ -230,7 +230,7 @@ Interestingly if the promise you throw doesn't resolve a value or resolves to an
 ## The bear trap
 
 Easy? Sounds it, but I'm going to let you into a secret - I've tried writing this before.
```

Comment:
I liked this section best. I was now onboard with (sort of) what we were doing and getting a bit worried about these references to the promise and following along more intently